tests/experimental/datasources/tasks.py

```diff
@@ -0,0 +1,36 @@
+"""
+CD to this directory to run these invoke commands.
+"""
+
+from __future__ import annotations
+
+import pathlib
+
+import invoke
+import pandas as pd
+
+TEST_ROOT = pathlib.Path(__file__).parent.parent.parent.resolve(strict=True)
+
+
+@invoke.task(aliases=["gen-assets"])
+def generate_asset_files(
+    ctx: invoke.Context,
+    source_csv_dir: str | pathlib.Path = None,
+    limit: int = 12,
+    year: str = 2019,
+):
+    """Generate or re-generate various test asset files."""
```

Comment:
@Kilo59 Are these files automatically cleaned up?  Thanks.

Reply:
No this is the utility I used to generate the test JSON files. See my longer explanation below.

https://github.com/great-expectations/great_expectations/pull/6780#discussion_r1087920374